### DISCUSSION TITLE
New Feature Ideas

### DISCUSSION THREAD

**Discussion by Aveek-Saha** [Aveek-Saha/MusicPlayer#52] [Ideas]
I am looking for new feature ideas for Dusk Player, keeping in line with the projects goal for being a minimalist Music Player.

If you want to contribute, please raise a new issue and if you want to work on it as well, wait for me to assign it to you. Once assigned, create a new Pull Request with your changes.

**Comment by michielst** [comment]
A feature I would want is a dynamic playlist directory.

This would mean that whenever a new song has been added to the filePath directory. Dusk Player would pick up on it and append it to the songs list withouth having to reopen the application.

**Comment by Aveek-Saha** [comment]
This could be useful, could you open a new issue with the feature request template?

**Comment by johnnyhliang** [comment]
Also, could we save playlists? Like you could set a playlist to a directory folder on your computer with all the music files.

**Comment by ghost** [comment]
@Aveek-Saha a whole new better UI without a menu bar using svelte-spa-router

**Reply by Aveek-Saha** [reply]
Can you send a screenshot of what menu bar you are referring to?

Also note that this is a discussion thread only for new features and not design changes.

**Comment by gregoftheweb** [comment]
I don't know if this is still an active project, but a feature I would like to see (and could assist with) would be a plain folder tree instead of a generated playlist.   So you could load a song or directory of songs or the player could understand to play the songs in "this" directory.
